keyboards/aidansmithdotdev/sango/readme.md

```diff
@@ -0,0 +1,19 @@
+# Sango
+
+![aidansmithdotdev/sango](https://github.com/Aidan-OS/Sango/blob/main/Sango.jpg)
```

Comment:
Please rehost this on Imgur.

Reply:
Are there any other websites it is acceptable to host images on? I have a particular distaste for imgur.